lang/ja.yml

```
@@ -83,110 +83,110 @@ INSPECTOR_TOGGLED: "検査官モードが{有効|無効}になりました。"
-NO_DATA_LOCATION: "番号 {データ|トランザクション|相互作用|メッセージ} 見つかった。"
+NO_DATA_LOCATION: "{データ|トランザクション|相互作用|メッセージ}は見つかりませんでした。"
 NO_PERMISSION: "あなたにはそれをする許可がありません。"
-NO_RESULTS: "結果が見つかりません。"
-NO_RESULTS_PAGE: "番号 {結果|データ} そのページで見つかりました。"
-NO_ROLLBACK: "番号 {保留中|前へ} ロールバック/復元が見つかりました。"
+NO_RESULTS: "結果が見つかりませんでした。"
+NO_RESULTS_PAGE: "このページには{結果|データ}は見つかりませんでした。"
+NO_ROLLBACK: "{保留中の|進んでいる}ロールバック/復元は見つかりませんでした。"
 PATCH_INTERRUPTED: "アップグレードが中断されました。再起動時に再試行します。"
-PATCH_OUTDATED_1: "古いデータベースをアップグレードできません {0}。"
-PATCH_OUTDATED_2: "CoreProtectがサポートされているバージョンでアップグレードしてください。"
-PATCH_PROCESSING: "新しいデータを処理しています。お待ちください..."
-PATCH_SKIP_UPDATE: "スキップ {テーブル|インデックス} {更新|作成|削除} の上 {0}。"
-PATCH_STARTED: "実行する {0} アップグレード。お待ちください..."
-PATCH_SUCCESS: "正常にアップグレードされました {0}。"
-PATCH_UPGRADING: "データベースのアップグレードが進行中です。お待ちください..."
-PLEASE_SELECT: "選んでください: \"{0}\" また \"{1}\" 。"
+PATCH_OUTDATED_1: "{0}は古いデータベースをアップグレードできません。"
+PATCH_OUTDATED_2: "CoreProtect がサポートされているバージョンにアップグレードしてください。"
+PATCH_PROCESSING: "新しいデータを処理しています。しばらくお待ちください..."
+PATCH_SKIP_UPDATE: "{0}の{テーブル|インデックス}の{更新|作成|削除}をスキップします。"
+PATCH_STARTED: "{0} のアップグレードを実行します。しばらくお待ちください..."
+PATCH_SUCCESS: "{0} は正常にアップグレードされました。"
+PATCH_UPGRADING: "データベースのアップグレードが進行中です。しばらくお待ちください..."
+PLEASE_SELECT: "選んでください: \"{0}\" または \"{1}\" 。"
 PREVIEW_CANCELLED: "プレビューがキャンセルされました。"
 PREVIEW_CANCELLING: "プレビューをキャンセルしています..."
 PREVIEW_IN_GAME: "ゲーム内でのみロールバックをプレビューできます。"
-PREVIEW_TRANSACTION: "プレビューできません {コンテナ|在庫} トランザクション。"
+PREVIEW_TRANSACTION: "{コンテナ|在庫}の入れ替えがプレビューできません"
 PURGE_ABORTED: "パージに失敗しました。データベースが破損している可能性があります。"
 PURGE_ERROR: "{0} のデータを処理できません"
-PURGE_FAILED: "パージに失敗しました。後でもう一度やり直してください。"
-PURGE_IN_PROGRESS: "進行中のパージ。後でもう一度やり直してください。"
-PURGE_MINIMUM_TIME: "分離できるのは、より古いデータのみです。({0}{日|時間})"
-PURGE_NOTICE_1: "これには時間がかかる場合がありますのでご注意ください。"
+PURGE_FAILED: "パージに失敗しました。もう一度やり直してください。"
+PURGE_IN_PROGRESS: "パージは進行中です。もう一度やり直してください。"
+PURGE_MINIMUM_TIME: "パージできるのは、より古いデータのみです。({0}{日|時間})"
+PURGE_NOTICE_1: "これには時間がかかる場合があります..."
 PURGE_NOTICE_2: "完了するまでサーバーを再起動しないでください。"
-PURGE_OPTIMIZING: "データベースの最適化。お待ちください..."
+PURGE_OPTIMIZING: "データベースの最適化をしています。しばらくお待ちください..."
 PURGE_PROCESSING: "{0} のデータを処理中..."
 PURGE_REPAIRING: "修復しようとしています。これには時間がかかる場合があります..."
-PURGE_ROWS: "{0}{行|行} のデータが削除されました。"
+PURGE_ROWS: "{0}行のデータが削除されました。"
 PURGE_STARTED: "データの削除は\"{0}\"で開始されました。"
-PURGE_SUCCESS: "データの分離に成功しました。"
-RELOAD_STARTED: "構成の再読み込み-しばらくお待ちください。"
-RELOAD_SUCCESS: "構成が正常に再ロードされました。"
+PURGE_SUCCESS: "データのパージに成功しました。"
+RELOAD_STARTED: "構成の再読み込みをしています。しばらくお待ちください。"
+RELOAD_SUCCESS: "構成が正常に再読み込みされました。"
 ROLLBACK_ABORTED: "ロールバックまたは復元が中止されました。"
-ROLLBACK_CHUNKS_FOUND: "見つかった {0} {チャンク|チャンク} 変更します。"
-ROLLBACK_CHUNKS_MODIFIED: "変更 {0}/{1} {チャンク|チャンク}。"
-ROLLBACK_COMPLETED: "{ロールバック|復元|プレビュー} のために完成{0}\" 。"
-ROLLBACK_EXCLUDED_USERS: "除外 {ユーザー|ユーザー}: \"{0}\" 。"
-ROLLBACK_INCLUDE: "{含まれる|除外される} {ブロック|エンティティ|ターゲット} {タイプ|タイプ}: \"{0}\" 。"
+ROLLBACK_CHUNKS_FOUND: "変更する{0}{チャンク|チャンク}が見つかりました。"
+ROLLBACK_CHUNKS_MODIFIED: "{0}/{1}{チャンク|チャンク} 変更"
```

```suggestion
ROLLBACK_CHUNKS_MODIFIED: "変更中 {0}/{1}{チャンク|チャンク}"
```

「変更中 2/5チャンク」 みたいな表記になったほうが自然と思う
I think it would be more natural to use something like “changing 2/5 chunks”